snapcraft/meta/snap_yaml.py

```diff
@@ -344,6 +342,28 @@ def _create_snap_app(app: App, assumes: Set[str]) -> SnapApp:
     return snap_app
 
 
+def _get_grade(grade: Optional[str], build_base: Optional[str]) -> str:
+    """Get the grade for a project.
+
+    If the build_base is `devel`, then the grade should be `devel`.
+    If the grade is not defined, default to `stable`.
```

Comment:
Is the current behaviour that the grade defaults to stable? (My intuition is that grade being undefined would always be devel, but obviously we shouldn't change behaviour unless we have a spec change.)